napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -0,0 +1,61 @@
+from copy import copy
+
+from ._labels_utils import interpolate_coordinates
+
+
+def pick(layer, event):
+    """Change the selected label to the same as the region clicked."""
+    # on press
+    layer.selected_label = layer._value or 0
+    yield
+
+    # on move
+    while event.type == 'mouse_move':
+        yield
+
+    # on release
+    layer._last_cursor_coord = None
+    layer._block_saving = False
+
+
+def paint(layer, event):
+    """Paint with the currently selected label."""
+    # on press
+    layer._save_history()
+    layer._block_saving = True
+    layer.paint(layer.coordinates, layer.selected_label)
+    layer._last_cursor_coord = copy(layer.coordinates)
```

Comment:
which of these private variables assigned to the layer can be made local to this function instead? also I would generally go for the `obj.copy()` method over `copy.copy(obj)` in most situations

Reply:
great observation - yes, this could be cleaned up - done now